pages/linux/sar.md

```diff
@@ -0,0 +1,27 @@
+# sar
+
+> Monitor performance of various Linux subsystems.
+
+- Report I/O and transfer rate issued to physical devices, every second (press CTRL+C to quit):
+
+`sar -b {{1}}`
+
+- Report a total of 10 network device statistics, one per 2 seconds:
+
+`sar -n DEV {{2}} {{10}}`
+
+- Report CPU utilization every 2 seconds (press CTRL+C to quit):
+
+`sar -u ALL {{2}}`
+
+- Report a total of 20 memory utilization statistics,  one per second:
+
+`sar -r ALL {{1}} {{20}}`
+
+- Report the run queue length and load averages:
+
+`sar -q {{1}} {{1}}`
```

Comment:
Does the first 1 mean its once per second, should we mention that if that is the case ?

Reply:
almost every command has the same structure, the first arg is the refresh granularity, and the second (if any) is the total number of stats to print. 
If the second arg is not provided, it assumes infinity.

So, I think this is obvious, and we should not write it explicitly to confuse people

Reply:
I got a bit confused because you mention 2 things - queue length and load average, and then there are 2 params. Its not very obvious whether the two "1"s are for granularity and no. of stats, or the granularity for queue length and load svg.

That's why I wanted to be extra clear in case there's any confusion. What do you think ?

Reply:
I think lets just skip the second param, if anyone wants, it is present in the man page.
This would make it less confusing.